packages/odyssey-react/src/components/List/DescriptionDetails.tsx

```diff
@@ -28,7 +29,7 @@ const DescriptionDetails = forwardRef<HTMLElement, DescriptionDetailsProps>(
 
     return (
       <dd {...omitProps} ref={ref} className={styles.details}>
-        {children}
+        <Text children={children} />
```

Comment:
I considered changing the type annotation here to only allow `ReactText` children but decided against a breaking change (or the complexity to support both APIs at once) and left the API open to composition if necessary